CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -17,6 +17,8 @@ internal class QuantityGenerator : GeneratorBase
         private readonly string _valueType;
         private readonly Unit _baseUnit;
 
+        private readonly string[] _decimalTypes = { "BitRate", "Information", "Power" };
```

Comment:
If it is not too much hassle, I would rather take a list of all quantities in the ctor to determine the value type from their `Quantity` type.

We are not likely to get more `decimal` types anytime soon, rather changing them to `double` I think, but it may take some time and this is one less place to maintain when that changes.

Reply:
This wasn't needed anymore since I reworked the codegen, so I removed it 👍